spec/requests/issues_controller_spec.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require "rails_helper"
+require "json"
```

Comment:
Is this needed after we load the Rails helper? 🤔

Reply:
Yes, we don't need it 👍